addons/hr_expense/static/src/views/expense_line_widget.js

```diff
@@ -26,10 +30,54 @@ export class ExpenseLinesListRenderer extends ListRenderer {
     }
 }
 
+export class ExpenseLinesKanbanRecord extends KanbanRecord {
+
+    setup() {
+        super.setup();
+        this.orm = useService("orm");
+    }
+
+    /** @override **/
+    async onGlobalClick(ev) {
+        const expenseName = this.props.record.data.name;
+        const attachments = await this.orm.call('hr.expense', 'get_expense_attachments', [this.props.record.resId]);
+        const files = attachments.map((attachment, index) => ({
+            isImage: true,
+            isViewable: true,
+            displayName: expenseName + `(${index + 1})`,
```

Comment:
This might not behave nicely in all languages though?
Maybe using the translation function would be better
```suggestion
            displayName: expenseName + ` (${index + 1})`,
```

Reply:
Why not? and how should I use the translation function?

Reply:
For right to left languages, they I believe they would expect the `(...)` on the other side (not 100% sure)

Reply:
I tested with Arabic, and it was added to the correct side 
![image](https://github.com/odoo/odoo/assets/160074741/3e08ed6d-5192-4b0c-8dc9-5e76e9d3dc61)